apps/frontend/app/home/home.module.css

```
@@ -27,10 +27,179 @@
 /* ========================= No setup section styles - END ===================================== */
 
 /* ========================= Shareable url section styles - START ===================================== */
-/* TODO: Add shareable-url rules here. The first rule has been partially set up for you */
 
 .shareableUrls {
   color: var(--white);
+  background-color: var(--dark);
+  padding-top: var(--spacer-9);
+  padding-bottom: var(--spacer-12);
 }
 
-/* ========================= Shareable url section styles - END ===================================== */
+.shareIcon {
+  display: block;
+  margin: auto;
+  text-align: center;
+  margin-bottom: var(--spacer-1);
+}
+
+.shareableUrls>p {
+  font-size: var(--larger);
+  text-align: center;
+}
+
+.shareableUrls>h2 {
+  font-size: calc(var(--xx-large) + var(--xx-large));
+  text-align: center;
+  margin-top: var(--spacer-1);
+}
+
+.slideContainer {
+  display: flex;
+  margin-top: var(--spacer-12);
+  gap: var(--spacer-12);
+}
+
+.slideContainer>img {
+  width: fit-content;
+  height: 70vh;
+}
+
+.trim {
+  width: fit-content;
+  height: 70vh;
+  min-width: 50vw;
+  overflow: hidden;
+  position: relative;
+}
+
+.animatedContainer {
+  width: fit-content;
+  height: 70vh;
+  min-width: 50vw;
+  overflow: hidden;
+  position: relative;
+  animation: fadeIn 3s;
+}
+
+.trim>img {
+  height: 100%;
+  min-width: fit-content;
+  position: absolute;
+  right: 0;
+}
+
+.animatedContainer>img {
+  height: 100%;
+  min-width: fit-content;
+  position: absolute;
+  right: 0;
+}
+
+.slideContainer>div {
+  width: 420px;
+  display: flex;
+  flex-direction: column;
+  gap: 65px;
+}
+
+.slideContainer>div>div {
```

Is there another way you can target this element? What if we decide to switch the child element from a `div` to another element?

```suggestion
.slideContainer > div > div {
```